src/site/stages/build/process-cms-exports/transformers/helpers.js

```diff
@@ -37,6 +76,28 @@ function getDrupalValue(arr) {
   return null;
 }
 
+/**
+ * A very specific helper function that expects to receive an
+ * object and an imageStyle string
+ * @param {object}
+ * @return {object}
+ */
+function getImageCrop(obj, imageStyle = null) {
+  if (imageStyle !== null) {
+    const imageObj = obj;
```

Comment:
To keep with the style we've got most other places in `vets-website` and ensure that we're not overriding `obj`, I'd recommend using either a new object literal and spread the properties of `obj` or `Object.assign()`.